src/mlbstandings/app.py

```
@@ -0,0 +1,52 @@
+#    if len(args) > 0:
+#        d = datetime(int(args[0]), 12, 31, 0, 0, 0, 0, ZoneInfo('Etc/UTC'))
+#        backfill = True
+#    else:
+    d = datetime.now(tz=ZoneInfo('Etc/UTC'))
+    # More scopes? Re-run gcloud auth application-default login.
+    # But not working locally :(
+    # How did I update scopes for the cloud function esp auth/drive ?
+    scopes = ['https://www.googleapis.com/auth/drive',  # to create spreadsheets
+              'https://www.googleapis.com/auth/spreadsheets',
+              'https://www.googleapis.com/auth/drive.metadata.readonly']  # unlikely we need this?
+    creds = google.auth.default(scopes=scopes)[0]  # type: ignore
```

Pylint (3.3.7)

[info] Module 'google' has no 'auth' member, but source is unavailable. Consider adding this module to extension-pkg-allow-list if you want to perform analysis based on run-time introspection of living objects.

(I1101)